src/components/MusicalTyping.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <div class="mx-auto text-left mt-64" style="width:700px">
+    <div>
+      <button class="w-8"></button>
+      <key-button v-for="key in sharpKeys" :key="key.key" :meta="key"></key-button>
+    </div>
+    <div>
+      <key-button v-for="key in naturalKeys" :key="key.key" :meta="key"></key-button>
+    </div>
+  </div>
+</template>
+
+<script>
+  import KeyButton from "./KeyButton";
+
+  export default {
+    name: "MusicalTyping",
+    components: {KeyButton},
+    props: ['octave'],
+    data() {
+      return {
+        activeFrequencies: {},
+        keys: [
+          {
+            letter: 'a',
+            pianoKey: 'C',
+            offset: 4,
```

Comment:
I see how we use the offset, but don't understand what the offset means. Could you explain?